sample-linux.ini

```diff
@@ -10,6 +10,10 @@ maxCacheEntries = 2000000
 mruSymbolStateFile = /tmp/snappy-mru-symbols.json
 maxMRUSymbolsPersist = 10
 
+[DiskCache]
+cachePath = /tmp/snappy/cache
+maxCacheFiles = 50
```

Comment:
`xul` and `libxul` seem to be outliers. They are 15MB size each, but most of the files are < 500KB.